src/widgets/SimpleTable.js

```diff
@@ -0,0 +1,186 @@
+/* eslint-disable react/no-unused-prop-types */
+/* eslint-disable react/forbid-prop-types */
+/**
+ * mSupply Mobile
+ * Sustainable Solutions (NZ) Ltd. 2019
+ */
+
+import React from 'react';
+import PropTypes from 'prop-types';
+
+import { StyleSheet, FlatList, View, Text } from 'react-native';
+import { ROW_BLUE, APP_FONT_FAMILY, SUSSOL_ORANGE } from '../globalStyles/index';
+
+/**
+ * Simple table component only rendering scalar values in a cell. Not intended
+ * for entering data. Below are frequently used props. Many other props are
+ * available for styling, documented in PropTypes.
+ * @prop    {array}  data    Array of data. Example below.
+ * @prop    {array}  columns Array of column objects. Example below.
+ * @prop    {string} title   Title for this table.
+ * data = [ { a: 1, b: 2}, {a: 3, b: 4}, .. ]
+ * columns = [ { key: 'a', width: 1, title: 'A COLUMN' }, { key: 'b', width: 1, title: 'B COLUMN' }]
+ */
+export class SimpleTable extends React.PureComponent {
+  keyExtractor = ({ index }) => index;
+
+  renderTitle = () => {
+    const { title } = this.props;
+    const { titleContainer, titleFont } = localStyles(this.props);
```

Comment:
as per our conversation, not a fan of recreating styles for every cell etc.. 
can leave this be as longs as you make an issue to refactor it somehow (it does look super clear and simple and I do like the neatness of the code). 
the only way i can think of refactoring at this stage (with the least impact on performance is, say for a cell: 
`cell = this.adjustStyle(cell, index, header, width)`
...
```
// when component mounts
this.styles = localStyles(this.props)
```
```
adjustmentStyles = ({index, width} = {}) = { 
cell: { flex: width, backgroundColor: index % 2 === 0 ? ROW_BLUE : 'white' })
```
...
```
adjustStyle = (type, ...extra) => {
   return { ...this.styles[type], ...this.adjustmentStyles(...extra)[type]}
}
```

Reply:
#885